SimPEG/EM/Analytics/NSEM.py

```diff
@@ -0,0 +1,128 @@
+import numpy as _np
```

Comment:
Thanks @lheagy  and @rowanc1 for the feedbacks. I'm learning a lot like this.
the higher level explicit import in Analytics __init__.py  was already done, so it did not fix the tab completion issue.
I just reverse all the _np import etc. and I set up the __all__ parameter but it does not fix the tab completion issue either (it seems is effect is mainly when we do  from __ import *). 
See below:
![__init__ py simpeg 2017-06-08 20-11-19](https://user-images.githubusercontent.com/13682747/26959758-abf39a50-4c86-11e7-947b-1d02add30829.png)